app/helpers/application_helper.rb

```diff
@@ -88,14 +88,17 @@ def pretty_json(raw_json_string)
   end
 
   def top_menu_items(format_values=nil)
-    tags = Rails.configuration.x.top_menu_items.collect do |menu_item|
-      (url = menu_item['url'] % format_values) if format_values
+    menu_items = [
+      { 'key' => 'monitoring', 'url' => '/dashboards/:monitoring' },
+      { 'key' => 'tuning', 'url' => '/dashboards/:tuning' }
+    ]
+    tags = menu_items.collect do |menu_item|
+      url = menu_item['url'] if format_values
       link_to(
         t("menu_item.#{menu_item['key']}"),
         (url || '#'),
-        id:     menu_item['key'],
-        class:  "submenu-item#{' disabled' unless url}",
-        target: ('_blank' if menu_item['target_new_window'])
+        id:    menu_item['key'],
+        class: "submenu-item#{' disabled' unless url}"
```

Comment:
What is the meaning of this `disabled`?
`url` exists always in this case

Reply:
in this case nope `      url = menu_item['url'] if format_values`

I think I will change the name since there are 2 names here

Reply:
see https://github.com/SUSE/blue-horizon/pull/11/files#diff-5f3fc5e3977d242572aa1d08551f5eb557de0ccaff30370838ee9df5386ea0daR96

Reply:
reverted